docs/docs/guides/integrations/authenticated-mongodb-charts.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Authenticated MongoDB Charts
+---
+
+This integration guide shows how you can embed authenticated MongoDB Charts in your web application using ZITADEL as authentication provider.
+
+## Setup ZITADEL Application
+
+Before you can embed an authenticated chart in your application, you have to do a few configuration steps in ZITADEL Console.
+You will need to provide some information about your app. We recommend creating a new app to start from scratch.
+
+1. Navigate to your [Project](https://console.zitadel.ch/projects)
+1. Add a new application at the top of the page.
+1. Select Web application type and continue.
+1. Use [Authorization Code](../../apis/openidoauth/grant-types#authorization-code) in combination with [Proof Key for Code Exchange (PKCE)](../../apis/openidoauth/grant-types#proof-key-for-code-exchange).
+1. Skip the redirect settings and confirm the app creation
+1. Copy the client ID, you will need to tell MongoDB Charts about it.
```

Comment:
```suggestion
6. Copy the client ID, you will need to tell MongoDB Charts about it.
```

Reply:
... you will need to configure MongoDB Charts with it.